### PR TITLE
DOC: Small whatsnew fixes

### PR DESCRIPTION
**Pull request by jschendel**
A couple small things I saw reading over the whatsnew notes.

### CODE REVIEW

File: doc/source/whatsnew/v0.25.0.rst [hunk 1]
```diff
@@ -1133,7 +1133,7 @@ Groupby/resample/rolling
 - Bug in :meth:`pandas.core.window.Rolling.median` and :meth:`pandas.core.window.Rolling.quantile` where MemoryError is raised with empty window (:issue:`26005`)
 - Bug in :meth:`pandas.core.window.Rolling.median` and :meth:`pandas.core.window.Rolling.quantile` where incorrect results are returned with ``closed='left'`` and ``closed='neither'`` (:issue:`26005`)
 - Improved :class:`pandas.core.window.Rolling`, :class:`pandas.core.window.Window` and :class:`pandas.core.window.EWM` functions to exclude nuisance columns from results instead of raising errors and raise a ``DataError`` only if all columns are nuisance (:issue:`12537`)
-- Bug in :meth:`pandas.core.window.Rolling.max` and :meth:`pandas.core.window.Rolling.min` where incorrect results are returned with an empty variable window`` (:issue:`26005`)
```
Comment (jbrockmendel) [inline doc/source/whatsnew/v0.25.0.rst]: Should “window” be ticked?

Comment (jschendel) [reply doc/source/whatsnew/v0.25.0.rst]: Yeah, I wasn't sure if the intention was to tick "window" or not.  The other instances of "window" in the whatsnew are not ticked, so I opted to leave this one unticked for consistency, but can tick it if that's be best course of action.

Comment (WillAyd) [reply doc/source/whatsnew/v0.25.0.rst]: Doesn't need to be ticked - just referring to the concept of a window not an object

**Review by WillAyd — APPROVED** [review]
lgtm

**Comment by WillAyd** [comment]
One other thing I noticed when browsing docs is the indentation of the section titled `Categorical dtypes are preserved during groupby` is wrong - can you fix that here?

**Comment by jreback** [comment]
@jschendel looks fine. merge when ready (if you are pushing again).

**Pull request merged**